contrib/get_supported_version_csi-sidecar.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 2023 The Kubernetes Authors.
```

Comment:
Some repos have codespell enabled as a github action which failed when this change was synced to the other repos, could you please review these errors?

```
Resulting CLI options  --check-filenames --skip ./.git,./.github/workflows/codespell.yml,.git,*.png,*.jpg,*.svg,*.sum,./vendor,go.sum,./release-tools/prow.sh
3
Error: ./release-tools/contrib/get_supported_version_csi-sidecar.py:35: ouputs ==> outputs
Error: ./release-tools/contrib/get_supported_version_csi-sidecar.py:101: relase ==> release
```

Reply:
addressed in https://github.com/kubernetes-csi/csi-release-tools/pull/229